example/configs/wasm.json

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "wasm",
+  "value": {
+    "chain-id": "constantine-3",
+    "nid": "1",
+    "rpc-url": "https://rpc.constantine.archway.io:443",
+
+    "home-dir": "/Users/sherpalden",
+
+    "keyring-backend": "os",
+    "key-name": "sherpalden",
```

Comment:
use keyring backend test. also, we won't need password after this.

Reply:
@izyak This might result in a security issue in a production environment while storing a plain unencrypted key. Better we should figure out how to automate the process with an encrypted key.